### PR TITLE
[ZH] Update `osu! wiki contribution guide`

### PR DESCRIPTION
**Pull request by cdwcgt**
check format

### CODE REVIEW

**Review by kj415j45 — DISMISSED** [review]

File: wiki/osu!_wiki_Contribution_Guide/zh.md [hunk 1]
```diff
@@ -1,50 +1,52 @@
...
+2. 当你添加、修改了文件以后，会有提示告诉当前分支"behind"（落后）, "ahead"（超前）或者 "even"（同步）于 `ppy:master`。如果提示说 `This branch is {n} commits behind ppy:master`（当前分支落后于 ppy:master 分支 n 个提交），或者`This branch is {x} commits ahead, {y} commits behind ppy:master`意味着你的分支已经过时了。
+
+   ![](img/fork-even.jpg "OK \(no commits and not outdated\)")
+
+   ![](img/fork-ahead.jpg "OK \(these are your commits\)")
+
+   ![](img/fork-behind.jpg "Potentially bad \(your branch is outdated\)")
 
-2\. 当你添加、修改了文件以后，会有提示告诉当前分支"behind"（落后）, "ahead"（超前）或者 "even"（同步）于 `ppy:master`。如果提示说 `This branch is {n} commits behind ppy:master`（当前分支落后于 ppy:master 分支 n 个提交），意味着你的分支已经过时了。
+   ![](img/fork-ahead-behind.jpg "Potentially bad \(your branch is outdated with your commits\)")
```
Comment (TPGPL) [inline wiki/osu!_wiki_Contribution_Guide/zh.md:43]: image tooltips should be translated

File: wiki/osu!_wiki_Contribution_Guide/zh.md [hunk 2]
```diff
@@ -53,54 +55,54 @@ outdated: true
...
+   对于文章翻译，在标题前面加上用方框括起来的两位语言代号，标题可以是你所翻译文章的名字。比如 `[FR] BBCode` 意味着 [BBCode](/wiki/BBCode) 这篇文章的法语翻译。
 
-9\. 填写描述框中所需要的信息，你可以删除没有使用的部分。
+9. 填写描述框中所需要的信息。你应该写上任何与 Pull Request 有关的信息，例如其完成程度已经你想让审阅者知道的任何信息。你也可以在描述中写入 "resolves #1" 或 "closes #1" 等关键字来让相关 issues 在这个 Pull Request 合并后自动关闭。（参见 Github Help 的文章 [Closing issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/)）
 
-10\. 完成后，点击 `Create pull request`。
+10. 完成后，点击 `Create pull request`。
 
-![](img/new-pull-request.jpg "Perfect")
+   ![](img/new-pull-request.jpg "Perfect")
```
Comment (TPGPL) [inline wiki/osu!_wiki_Contribution_Guide/zh.md:96]: remove the tooltip

**Review by TPGPL — COMMENTED** [review]
should also check the article for interpunction consistency, as there were some regular dots at the beginning

File: wiki/osu!_wiki_Contribution_Guide/zh.md (hunk 2 above)
Comment (cdwcgt) [reply wiki/osu!_wiki_Contribution_Guide/zh.md:96]: oh I forget one space here maybe

File: wiki/osu!_wiki_Contribution_Guide/zh.md [hunk 3]
```diff
@@ -1,50 +1,52 @@
...
+2. [登录](https://github.com/login)你的账号。
+3. 下一步：[服务端的仓库克隆](#服务端的仓库克隆)。
 
 ### 服务端的仓库克隆
 
-1\. 进入 [`osu-wiki` 仓库](https://github.com/ppy/osu-wiki)。
+1. 进入 [`osu-wiki` 仓库](https://github.com/ppy/osu-wiki)。
 
-2\. 点击 `Fork`.
+2. 点击 `Fork`.
```
Comment (TPGPL) [inline wiki/osu!_wiki_Contribution_Guide/zh.md:19]:
```suggestion
2. 点击 `Fork`。
```

File: wiki/osu!_wiki_Contribution_Guide/zh.md [hunk 4]
```diff
@@ -1,50 +1,52 @@
...
+3. 再次点击 `Fork`，进入你自己的仓库。
 
-4\. 看到如下图所示的内容，意味你已经克隆了 `ppy/osu-wiki` 仓库，并且处在你自己的仓库中。
+4. 看到如下图所示的内容，意味你已经克隆了 `ppy/osu-wiki` 仓库，并且处在你自己的仓库中。
 
-![](img/forked.jpg)
+   ![](img/forked.jpg)
 
-3\. 下一步：[同步你的仓库](#同步你的仓库).
+5. 下一步：[同步你的仓库](#同步你的仓库).
```
Comment (TPGPL) [inline wiki/osu!_wiki_Contribution_Guide/zh.md:29]:
```suggestion
5. 下一步：[同步你的仓库](#同步你的仓库)。
```

File: wiki/osu!_wiki_Contribution_Guide/zh.md [hunk 5]
```diff
@@ -53,54 +55,54 @@ outdated: true
...
 
-**当你阅读完成 *GitHub 网页* 或 *GitHub Desktop*，你可以继续阅读下一章节。**
+**当你阅读 *GitHub 网页* 或 *GitHub Desktop*文章后，你可以继续阅读下一部分。**
 
 ## 完成
 
 ### 开一个 Pull Request
 
-1\. 进入 [`ppy/osu-wiki` 仓库](https://github.com/ppy/osu-wiki).
+1. 进入 [`ppy/osu-wiki` 仓库](https://github.com/ppy/osu-wiki).
```
Comment (TPGPL) [inline wiki/osu!_wiki_Contribution_Guide/zh.md:66]:
```suggestion
1. 进入 [`ppy/osu-wiki` 仓库](https://github.com/ppy/osu-wiki)。
```

File: wiki/osu!_wiki_Contribution_Guide/zh.md [hunk 6]
```diff
@@ -53,54 +55,54 @@ outdated: true
...
+   ![](img/head-fork.jpg)
 
-![](img/head-fork.jpg)
+6. 点击 `compare` 下拉菜单，选择你已经创建的那个分支，分支一般按字母排序。
 
-6\. 点击 `compare` 下拉菜单，选择你已经创建的那个分支，分支一般按字母排序。
+   ![](img/compare-branch.jpg)
 
-![](img/compare-branch.jpg)
+7. 点击 `Create pull request`.
```
Comment (TPGPL) [inline wiki/osu!_wiki_Contribution_Guide/zh.md:86]:
```suggestion
7. 点击 `Create pull request`。
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**